release-engineering/role-handbooks/branch-manager.md

```diff
@@ -503,30 +507,43 @@ The `variants.yaml` config file, used in conjunction with the GCB Builder, allow
 
 Update the [variants for the `kubekins-e2e` image](https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/variants.yaml).
 
-- Remove the oldest release variant
+- Remove the release variants for the unsupported releases
 - Add an entry for the newest release variant
-- Ensure the following:
+- Copy the `master` variant, rename it to the newest release and ensure the following:
   - The `K8S_RELEASE` marker for `experimental` matches `master`
+  - The `CONFIG` marker for the newest release matches the variant/release name
   - The `K8S_RELEASE` marker for the newest release variant is `latest-x.y`
   - The `K8S_RELEASE` marker for every other release variant is `stable-x.y`
 
-Create a PR with this change and wait for it to be merged ([example PR](https://github.com/kubernetes/test-infra/pull/15015)).
+Create a PR with this change and wait for it to be merged ([example PR](https://github.com/kubernetes/test-infra/pull/20076)).
 
 **Wait for the `test-infra-push-kubekins-e2e` presubmit to finish (you can [check on prow](https://prow.k8s.io/?job=post-test-infra-push-kubekins-e2e)).**
 
 **`git pull` latest from `master` before continuing.**
 
 #### Generate release branch jobs
 
+This task should be done after the release is complete and previous PRs are merged. The following steps should be run from the test-infra repository's root.
+
 1. Generate the new release branch jobs:
 
    ```shell
    bazel run //releng:prepare_release_branch
    ```
 
+   Alternatively, if you want to run Bazel in a container, you can use the command such as:
+
+   ```shell
+   docker run -it \
+      -v $PWD:/workspace/src/k8s.io/test-infra \
+      -v $HOME/.cache/bazel:/root/.cache/bazel \
+      -w /workspace/src/k8s.io/test-infra \
+      gcr.io/k8s-testimages/launcher.gcr.io/google/bazel:v20201028-8000225-test-infra run //releng:prepare_release_branch
```

Comment:
Let's add a hint that the container image tag will be regularly updated in test-infra and how to obtain the latest version of it.

Reply:
@saschagrunert Good point. What is the best way to obtain the latest version of the image?

Reply:
According to https://console.cloud.google.com/gcr/images/k8s-testimages/GLOBAL/launcher.gcr.io/google/bazel?gcrImageListsize=30 this image should refer to the latest one:
```
gcr.io/k8s-testimages/launcher.gcr.io/google/bazel:latest-test-infra
```

Reply:
@saschagrunert Thank you. Updated the example to refer to the latest image. Is that enough?